package.json

```diff
@@ -38,14 +38,14 @@
     "@hot-loader/react-dom": "^16.8.6",
     "@khanacademy/flow-to-ts": "^0.1.9",
     "@microsoft/api-extractor": "^7.13.0",
-    "@storybook/addon-a11y": "^6.3.8",
-    "@storybook/addon-actions": "^6.3.8",
-    "@storybook/addon-docs": "^6.3.8",
-    "@storybook/addon-essentials": "^6.3.8",
-    "@storybook/addon-links": "^6.3.8",
-    "@storybook/addons": "^6.3.8",
-    "@storybook/react": "^6.3.8",
-    "@storybook/theming": "^6.3.8",
+    "@storybook/addon-a11y": "^6.4.3",
+    "@storybook/addon-actions": "^6.4.3",
+    "@storybook/addon-docs": "^6.4.3",
+    "@storybook/addon-essentials": "^6.4.3",
+    "@storybook/addon-links": "^6.4.3",
+    "@storybook/addons": "^6.4.3",
+    "@storybook/react": "^6.4.3",
```

Comment:
hey, did you have some compatibility issues with 6.4.9?

Reply:
No, I didn't check if newer version exists. I will bump it to 6.4.9

Reply:
Look like everything works on 6.4.9 :)